BUILD.bazel

```diff
@@ -1,6 +1,11 @@
 load("@bazel_gazelle//:def.bzl", "gazelle")
 load("@io_bazel_rules_go//go:def.bzl", "TOOLS_NOGO", "nogo")
 
+exports_files([
```

Comment:
add a comment why these files are "leaked" from the usual Bazel package encapsulation

Reply:
Comment added